zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchExporterConfiguration.java

```diff
@@ -142,6 +142,8 @@ public boolean shouldIndexValueType(final ValueType valueType) {
         return index.user;
       case AUTHORIZATION:
         return index.authorization;
+      case TENANT:
```

Comment:
@Ben-Sheppard @Oleksiivanov We need to keep supporting the old exportes as Optimize is not yet migrating to the new `CamundaExporter`. So we shouldn't remove this.

Reply:
I double-checked if we should support new records in the old exporters. The conclusion is "we should export records with the old exporter only if they are needed/consumed". Since these will not be consumed by Optimize we should be able to get rid of them anyways.